Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -42,6 +42,7 @@ extension XCScheme {
         public var launchStyle: Style
         public var askForAppToLaunch: Bool?
         public var pathRunnable: PathRunnable?
+        public var customWorkingDirectory: Path?
```

Comment:
Taking a look at the codebase for any serialised properties `String` has been used in the past whereas `Path` was only convenience within some of the APIs before bridging to `String` 

For example:

https://github.com/tuist/XcodeProj/blob/6943d5c5f9085d070dbde3c669144e30d41ec635/Sources/XcodeProj/Scheme/XCScheme%2BLaunchAction.swift#L70

https://github.com/tuist/XcodeProj/blob/6943d5c5f9085d070dbde3c669144e30d41ec635/Sources/XcodeProj/Objects/Files/PBXFileElement.swift#L12

https://github.com/tuist/XcodeProj/blob/6943d5c5f9085d070dbde3c669144e30d41ec635/Sources/XcodeProj/Objects/BuildPhase/PBXCopyFilesBuildPhase.swift#L22

If I'm not mistaken, Xcode may also accept variables in this field too as such it's not exclusively `Path`s :/ 

![Screenshot 2022-09-02 at 07 55 19](https://user-images.githubusercontent.com/11914919/188076942-75288e3e-65e9-426f-8d23-d53dadbbcd3e.png)